backend/cfg_selectgen.ml

```diff
@@ -654,11 +654,12 @@ module Make (Target : Cfg_selectgen_target_intf.S) = struct
               | Maybe_out_of_range ->
                 Target.is_store_out_of_range chunk ~byte_offset:!byte_offset
             in
-            let new_addressing_mode =
+            let _ =
```

Comment:
Is the binding needed?
(If we want to keep it, I
think it would be useful
to have a type annotation.)

Reply:
Fixed.